### DISCUSSION TITLE
Use ansible.cfg to get ANSIBLE_COLLECTIONS_PATH and ANSIBLE_ROLES_PATH

### DISCUSSION THREAD

**Discussion by chancez** [ansible/ansible-lint#1327] [Ideas]
Currently in the latest release of ansible-lint, it's now trying to install my roles from requirements.yaml, which is cool, but i have all my stuff vendored into `collections/` and `roles/` currently for a few projects, and that's specified in `ansible-cfg`, and galaxy obeys that. So it would be great if `ansible-lint` did the same.

**Comment by joren485** [comment]
https://github.com/ansible-community/ansible-lint/discussions/1353#discussioncomment-367465 might be relevant to this idea.

**Reply by xenlo** [reply]
I get the same unexpected behavior. So I created a bug report: https://github.com/ansible-community/ansible-lint/issues/1364
